docs/make.jl

```diff
@@ -13,7 +13,7 @@ example_destination = joinpath(@__DIR__, "src", "examples")
 example_doc_paths = ["examples/$name.md" for name in example_names]
 
 for source in example_sources
-    Literate.markdown(source, example_destination; execute, documenter=true)
+    Literate.markdown(source, example_destination; documenter=true)
```

Comment:
Per `Literate.jl` docs:

> Since Documenter executes and captures results of `@example` block it is not necessary to use `execute=true` for markdown output that is meant to be used as input to Documenter.